### DISCUSSION TITLE
Damage model

### DISCUSSION THREAD

**Discussion by DP52** [peridigm/peridigm#321] [General]
What damage model can be used besides Critical Stretch?

When I do crack propagation problems, using elastic materials and Critical Stretch model, the crack can grow normally. But when the material is replaced by other materials, the crack is only open and does not expand, so I am thinking about whether it is a damage model. However, other damage models cannot be used normally.

The following is an error when using the Interface Aware model.
[]
![图片1](https://user-images.githubusercontent.com/128480691/236599424-b5294d33-94c7-4dad-b06e-b7c3d108e0ca.png)
(url)

In addition, when I used the Critical Energy model, I was prompted that this model was not available, as shown in the figure below.
[]
![图片2](https://user-images.githubusercontent.com/128480691/236599433-e9690a86-2b01-45eb-98ec-8dd09a9ab5de.png)
(url)

How to solve the problem that the crack only opens but does not expand?

**Comment by IntePNum** [comment]
I cannot see your picture

**Reply by IntePNum** [reply]
When I do crack propagation problems, using "elastic materials and Critical Stretch model", why the crack can grow normally in my model? it must set time-step quite quite small, i can only open? Can I see your model and yaml files?

**Reply by DP52** [reply]
Using elastic materials, cracks can be extended, but when I use elastic Correspondence materials and Critical Stretch model, the crack is only open and does not expand. 
The model and yaml files are as follows :
[
[wan.zip](https://github.com/peridigm/peridigm/files/11551451/wan.zip)
](url)
